portfolio.html

```diff
@@ -4,8 +4,42 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+    />
     <link rel="stylesheet" href="style.css" />
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
```

Comment:
Your project looks very dissimilar from the template design.

<img width="298" alt="Screen Shot 2022-08-18 at 8 47 37" src="https://user-images.githubusercontent.com/82426682/185410895-908965a0-3db4-43b4-97a9-21aacffd085b.png">

<img width="193" alt="Screen Shot 2022-08-18 at 8 47 54" src="https://user-images.githubusercontent.com/82426682/185410950-16e2ac8c-1170-4c7f-a959-2479476fc3b8.png">

Please add all the necessary details to make it as identical as possible.

<img width="555" alt="Screen Shot 2022-08-18 at 8 48 25" src="https://user-images.githubusercontent.com/82426682/185411068-8f49cd24-70a3-418b-be0b-568fcc9eb429.png">

Reply:
All the necessary details added, and also bug fixed